lib/drives/pages/drives_page.dart

```diff
@@ -19,14 +22,20 @@ class _DrivesPageState extends State<DrivesPage> {
         child: Text('Drives'),
       ),
       floatingActionButton: FloatingActionButton(
-        onPressed: () {
-          Navigator.of(context).push(
-            MaterialPageRoute(builder: (context) => const CreateDrivePage()),
-          );
-        },
+        onPressed: onPressed,
         backgroundColor: Theme.of(context).colorScheme.primary,
         child: const Icon(Icons.add),
       ),
     );
   }
+
+  void onPressed() async {
```

Comment:
did you want that to direct to the detailview and change it before the merge? because right now it's not possible to check whether the deletebutton works.